localauth0.toml

```diff
@@ -20,3 +20,8 @@ permissions = ["audience1:permission1", "audience1:permission2"]
 [[audience]]
 name = "audience2"
 permissions = ["audience2:permission2"]
+
+[access_token]
+custom_claims = [
```

Comment:
for instance I need to add to access_token a field "http://prima.it/user_uuid" because we have a custom actions on auth0 for generating uuid for user

Reply:
I'm just a bit confused on how should you use that value to test in an env if that uuid is always the same...

Reply:
we're not actually using localauth0 in an env (on qa and staging we use real auth0 on a staging tenant). We're using localauth0 in rust tests on CI, in order to avoid using functions mocks.

Reply:
Isn't it what it's been created for?